packages/api-client/src/team/team/TeamAPI.ts

```diff
@@ -78,10 +78,11 @@ export class TeamAPI {
     return response.data;
   }
 
-  public async deleteTeam(teamId: string, password: string): Promise<void> {
+  public async deleteTeam(teamId: string, password: string, verificationCode?: string): Promise<void> {
     const config: AxiosRequestConfig = {
       data: {
         password,
+        ...(verificationCode && {verification_code: verificationCode}),
```

Comment:
```suggestion
        verification_code: verificationCode,
```

Or is it a problem when sending an undefined verification code?

Reply:
ive just double checked, it is not a problem (according to the BE), i thought in general it would be better to not pass one if one is not required

Reply:
True, we just never did it this way. But either way is fine 🙂